packages/pn-pa-webapp/src/pages/NotificationDetail.page.tsx

```diff
@@ -49,24 +49,12 @@ const NotificationDetail = () => {
   const legalFactDownloadUrl = useAppSelector(
     (state: RootState) => state.notificationState.legalFactDownloadUrl
   );
-  const unfilteredDetailTableRows: Array<{
-    label: string;
-    rawValue: string | undefined;
-    value: ReactNode;
-  }> = [
+  const detailTableRows: Array<NotificationDetailTableRow> = [
```

Comment:
Fare molta attenzione quando si risolvono i conflitti.
In questo caso si sono perse le modifiche apportate da un'altra pr e in più il codice ha degli errori.
Riportare il file a come è su develop e applicare le modifiche atte a risolvere il bug oggetto di questa pr.
Per semplicità si può scaricare l'estensione Git History Diff di visual studio, e confrontare questo file con quello presente su develop.
Ogni volta che ci sono dei conflitti e  questi vengono risolti, rilanciare i test e l'applicazione per essere certi non ci siano errori